scheduler/scripts/run_main.py

```diff
@@ -111,9 +114,10 @@ def main(*,
             selector.update_cc_and_iq(site, cc_value, iq_value)
 
         for site in collector.sites:
-            # The starting twilight for the night for the site.
+            # Initial values until the evening twilight is executed, and a plan is made.
             night_start: Optional[datetime] = None
             night_done = False
+            plans = None
```

Comment:
This confirms what Bryan was saying about the plans being done one site at it time. Even if they have access at each other in the Selection phase we do it one output at time.